crates/consensus/consensus/src/lib.rs

```diff
@@ -414,6 +416,15 @@ pub enum ConsensusError {
         child_gas_limit: u64,
     },
 
+    /// Error indicating that the block gas limit is above the allowed maximum.
+    ///
+    /// This error occurs when the gas limit is more than the specified maximum gas limit.
+    #[error("child gas limit {block_gas_limit} is above the maximum allowed limit ({MAXIMUM_GAS_LIMIT_BLOCK})")]
```

Comment:
since this rule can be enforced on a standalone header, we can rephrase this a bit, maybe just `header gas limit...`

Reply:
should be changed